_gsocproposals/2018/proposal_LHCbHDF5.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Integration of the HDF5 file format within the LHCb analysis framework
+layout: gsoc_proposal
+project: LHCb
+year: 2018
+organization: CERN
```

Comment:
please put in all participating organizations. You can provide multiple ones as done e.g. [here](https://raw.githubusercontent.com/HEP-SF/hep-sf.github.io/master/_gsocproposals/2018/proposal_ATLASYampl.md).

Reply:
OK. Should be OK now.